docs/authorization.md

```diff
@@ -0,0 +1,80 @@
+
+## Registering a study
+
+Before any data can be uploaded for a study, a PCGL admin must register the study in the system via the `\study` endpoint. Only the study id is required - this should be provided by the study and must be unique within the PCGL. For CPHI projects, we have agreed on the format of the study id in advance. 
+
+The endpoint also takes an optional list of data submitters (users who can add / edit data for this study). 
+
+The `study` endpoint does not have an update option, so to modify the list of data submitters, an admin has to GET the study and then re-POST it with the existing + new information. 
+
+CAUTION: A POST request to the ingest/program replaces any existing program registration data for that program. 
+
+Once the PCGL internal identifer service is deployed, the study creation step will also call the identifier service to generate the internal identifier. This ensures that both IDs are recorded at the point of study creation. The quirk with allowing updates to study registrations will need to be fixed before then to avoid re-generating the internal ID. 
+
+## Authorization queries
+
+It is the responsibility of PCGL services to ensure that data is not released unless the user is authorized. The authz `allowed` endpoint is the primary interface for asking if a user can perform a specified action on a study. When calling `allowed` a service provides the following information, along with user information in the Bearer token:
+
+```
+{
+  "action": {
+    "endpoint": "string",
+    "method": "GET"
+  },
+  "studies": [
+    "string"
+  ]
+}
+```
+
+
+The action must match one of the endpoint / method combinations provided during [service registraion](/docs/service-registration.md).
+
+The authx service returns `True` or `False` depending on whether the given user can perform the action on the study/studies. It also logs a decision log as an audit trail that details the logic of the decision. 
+
+PCGL services should prioritize this endpoint for authorization decisions vs writing their own logic about data authorization given user and / or study information returned by the study and user endpoints. Why?
+
+* consistent authorization decision-making across the platform
+* allows for changes to only happen in one place (e.g. creating a new user role), rather than changing logic in multiple services
+
+
+## Adding study authorization for a user following a DACO approval 
+
+When a user has been approved by the DACO for access to a study, a PCGL Admin (and eventually a DACO approver once that role is defined) can add authorization for that user by providing the following info via POST to the `\user\{pcgl_id}` endpoint. 
+
+```
+{
+  "study_id": "string",
+  "start_date": "string",
+  "end_date": "string"
+}
+```
+
+You can look up the pcgl_id in CILogon, or search for the user by email with the `\user\lookup` endpoint. 
```

Comment:
it seems that the \user\lookup endpoint returns a comanage_id - is that the same as the pcgl_id? should it be the pcgl id instead?